cmd/tether/tether_linux.go

```diff
@@ -176,7 +189,7 @@ func (t *osopsLinux) backchannel(ctx context.Context) (net.Conn, error) {
 	}
 
 	// HACK: currently RawConn dosn't implement timeout so throttle the spinning
-	ticker := time.NewTicker(1000 * time.Millisecond)
+	ticker := time.NewTicker(10 * time.Millisecond)
```

Comment:
Helpfully there's a comment there as to why.... LOL!  :trollface: 

I'll add one now.